cpp/include/cudf/detail/utilities/algorithm.cuh

```diff
@@ -25,4 +29,47 @@ __device__ __forceinline__ T accumulate(Iterator first, Iterator last, T init, B
   }
   return init;
 }
+
+/**
+ * @brief Utility for calling thrust::copy_if.
+ *
+ * Workaround for thrust::copy_if bug (https://github.com/NVIDIA/thrust/issues/1302)
+ * where it cannot iterate over int-max values `distance(first,last) > int-max`
+ * This calls thrust::copy_if in 2B chunks instead.
+ */
```

Comment:
Can you please update docs to include `@tparam`, `@param` and `@return`?

Reply:
Done.